docker/env/version

```diff
@@ -1 +1 @@
-0.51
+0.51-py3.2
```

Comment:
we are ready to merge, please set to correct version

Reply:
update the version to 0.52